test/sql-luatest/coalesce_test.lua

```diff
@@ -0,0 +1,78 @@
+local server = require('luatest.server')
+local t = require('luatest')
+
+local g = t.group()
+
+g.before_all(function()
+    g.server = server:new({alias = 'master'})
+    g.server:start()
+end)
+
+g.after_all(function()
+    g.server:stop()
+end)
+
+g.test_coalesce_1 = function()
```

Comment:
name of the test does not tell us very transparently what is going on.

`test_something_1`, `test_something_2` is a bad practice.
Simple template:

`test_<what>_<when>_<additional condition>`